morse.rb

```diff
@@ -0,0 +1,54 @@
+MORSE = {
+  'a' => '.-',
+  'b' => '-...',
+  'c' => '-.-.',
+  'd' => '-..',
+  'e' => '.',
+  'f' => '..-.',
+  'g' => '--.',
+  'h' => '....',
+  'i' => '..',
+  'j' => '.---',
+  'k' => '-.-',
+  'l' => '.-..',
+  'm' => '--',
+  'n' => '-.',
+  'o' => '---',
+  'p' => '.--.',
+  'q' => '--.-',
+  'r' => '.-.',
+  's' => '...',
+  't' => '-',
+  'u' => '..-',
+  'v' => '...-',
+  'w' => '.--',
+  'x' => '-..-',
+  'y' => '-.--',
+  'z' => '--..'
+}.freeze
+
+def decode_char(char)
+  MORSE.each do |key, value|
+    return key.upcase if value == char
+  end
+end
+
+def decode_word(word)
+  new_word = ''
+  new_arr = word.split
+  new_arr.each do |value|
+    new_word += decode_char(value)
+  end
+  new_word
+end
+
+def decode_sentence(text)
+  new_sentence = ''
+  new_arr = text.split('   ')
+  new_arr.each do |value|
+    new_sentence += "#{decode_word(value)} "
+  end
+  new_sentence
+end
```

Comment:
- [ ] There is a trailing space from what this method returns. Kindly add `.strip` to `new_sentence` that is being returned from this method to remove the trailing space. Hint; `new_sentence.strip`

<img width="162" alt="chrome_PJAmHmLZlV" src="https://user-images.githubusercontent.com/81468675/163559883-ad951604-2a18-4518-b7f1-e30b703122dd.png">

Reply:
Thank you for the feedback.